control-plane/stor-port/src/types/v0/transport/volume.rs

```diff
@@ -932,3 +933,24 @@ impl CreateSnapshotVolume {
         &self.volume_params
     }
 }
+
+impl EventWithMeta for VolumeState {
```

Comment:
This should probably be done in the Agents

Reply:
Added it now in agents.